1001-2000/LOC1741Balaza.xml

```diff
@@ -42,14 +42,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: place</change>
+         <change who="JK" when="2024-07-18">Changed transcription, added fidal</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
             <place type="town">
-               <placeName>Bäläza</placeName>
+               <placeName xml:lang="gez" xml:id="n1">በለዛ</placeName>
```

Comment:
We can find a little of more detailed information on Balaza, in terms of historical geography (it being part of ...), administrative embedding and even coordinates, and sometimes get things like here: https://www.youtube.com/watch?v=1gbea-aEeUo. Could you please add some details? (with whatever place name I deal, I routinely type it into search engive and see what happen).

Reply:
I have added the administrative region, for which there was already a LOC, but I have not created a LOC for the subregion, ok?

Reply:
Please create. It is the same as we do for Ethiopian regions. It is actually ሰረጀቓ, ሰረጀቃ (I have never heard of it, and not in the EAE (strange!), but quite present) is alternative spelling;administratively it is  ንኡስ ዞባ.